.github/workflows/smoke-test.yml

```diff
@@ -1,7 +1,6 @@
-name: Manual test workflow
+name: CMS dev server test
 
-on:
-  workflow_dispatch:
+on: push
```

Comment:
I assume this will make these smoke tests run on every new set of PR commits and will show an ugly X if something fails. Should double check that after this merges.